manim/mobject/svg/text_mobject.py

```diff
@@ -287,7 +288,7 @@ def text2svg(self):
 
         if self.font == '':
             if NOT_SETTING_FONT_MSG != '':
-                print(NOT_SETTING_FONT_MSG)
+                logger.warning(NOT_SETTING_FONT_MSG)
```

Comment:
we may want to consider turning other error messages into constants as well.

Reply:
Agreed.